rust/common-tracing-macros/Cargo.toml

```diff
@@ -0,0 +1,13 @@
+[package]
+name = "common-tracing-macros"
```

Comment:
I wonder if this should just be a `common-macros` crate with a `tracing` feature. WDYT?

Reply:
Yeah, if we'll need more proc macros, doesn't make sense to itemize them per feature

Reply:
Reconfigured `common-tracing-macros` => `common-macros`